src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -62,4 +66,46 @@ public Entity getEntityFromDatastore(String entityName, String entityPropertyTit
     Entity entity = resultsList.get(0);
     return entity;
   }
+
+  /**
+  * Adds a game id to a user's list of games
+  */
+  public static boolean addGameToUser(Entity userEntity, DatastoreService datastore, String gameId) {
+ 
+    if(gameId == ""){
+        return false;
+    }
```

Comment:
Every spot where the function can return false is not what we want right? So should there be a log.severe() in each of these places?

Reply:
I added a few more